src/indexer/services/monitoringService/notificationSender/index.ts

```diff
@@ -0,0 +1,20 @@
+import { PublishCommand, PublishCommandInput, SNSClient } from "@aws-sdk/client-sns"
+import { logger } from "../../../../utils/logger"
+
+export class NotificationSender {
+  private snsClient: SNSClient
+
+  constructor(region: string) {
+    this.snsClient = new SNSClient({ region })
+  }
+
+  public async sendNotification(message: PublishCommandInput): Promise<void> {
+    try {
+      logger.debug("Sending notification Message: " + JSON.stringify(message))
```

Comment:
```suggestion
      logger.debug(`Sending notification Message: ${JSON.stringify(message)}`)
```

Use string interpolation to keep codebase consistent.